composer.json

```diff
@@ -19,11 +19,11 @@
         "php": "^7.1"
     },
     "require-dev": {
-        "ext-phar":                  "*",
         "ext-pdo":                   "*",
+        "ext-phar":                  "*",
+        "phpbench/phpbench":         "^0.13.0",
```

Comment:
@bendavies to avoid not seeing different version ranges when installing `phpbench` or `php_codesniffer`, I suggest adding an explicit `composer require` CLI statement in the respective pipeline entries

Reply:
@Ocramius do you mean to require a specific version of the deps?

Reply:
No, I mean adding a `composer require` line to the `install` step of the `.travis.yml` build pipeline entries

Reply:
done. i'm not sure i know what you mean by `to avoid not seeing different version ranges` though...

Reply:
@bendavies the point is that this lib is to be tested with all possible dependency ranges, so having a lot of dev dependencies means restricting those tested ranges. That's why I asked to move it from `require-dev` to a CLI command ;-)